tools/nni_cmd/config_schema.py

```diff
@@ -76,7 +76,7 @@ def setPathCheck(key):
             'optimize_mode': setChoice('optimize_mode', 'maximize', 'minimize'),
         },
         Optional('includeIntermediateResults'): setType('includeIntermediateResults', bool),
-        Optional('gpuNum'): setNumberRange('gpuNum', int, 0, 99999),
+        Optional('gpuIndices'): Or(int, And(str, lambda x: len([int(i) for i in x.split(',')]) > 0), error='gpuIndex format error!'),
```

Comment:
It's better not to remove gpuNum in this phase, and provide a warning message to let user know this breaking change.

Reply:
good suggestion, fixed.